src/pharmpy/tools/run.py

```diff
@@ -920,42 +920,34 @@ def is_strictness_fulfilled(res: ModelfitResults, statement: str) -> bool:
         unwanted_args = ['and', 'or', 'not']
         find_all_words = re.findall(r'[^\d\W]+', statement)
         args_in_statement = [w for w in find_all_words if w not in unwanted_args]
+        find_all_non_allowed_operators = re.findall(r"[^\w\s\.\<\>\=\!\(\)]", statement)
+        if len(find_all_non_allowed_operators) > 0:
+            raise ValueError(
+                f"Unallowed operators found: {', '.join(find_all_non_allowed_operators)}"
+            )
 
         # Check that only allowed arguments are in the statement
         if not all(map(lambda x: x in allowed_args, args_in_statement)):
             raise ValueError(
                 f'Some expressions were not correct. Valid arguments are: {allowed_args}'
             )
         else:
-            minimization_successful = res.minimization_successful
-            rounding_errors = res.termination_cause == "rounding_errors"
-            maxevals_exceeded = res.termination_cause == "maxevals_exceeded"
-            sigdigs = ArrayEvaluator([res.significant_digits])
+            minimization_successful = res.minimization_successful  # noqa
```

Comment:
Why is # noqa needed now?

Reply:
because otherwise we will get a linting error saying that the variables are never used.

Reply:
ok, now I see that the implementation was also changed. Then this makes sense